src/decoder_allocators.hpp

```diff
@@ -30,7 +30,7 @@
 #ifndef ZEROMQ_DECODER_ALLOCATORS_HPP
 #define ZEROMQ_DECODER_ALLOCATORS_HPP
```

Comment:
Indeed, the names are reserved. Unfortunately, they are used everywhere else, and this calls for keeping it consistent and fixing consistently everywhere at once, I think.